app/views/laundry_rooms/index.html.erb

```diff
@@ -13,32 +13,45 @@
 <% end %>
 <%= link_to 'Calendar', calendar_index_path %>
 
-<h1>Book a LaundryTime</h1>
-<table>
-  <tr>
-    <th>ID</th>
-    <th>Slots</th>
-    <th>Actions</th>
-  </tr>
-  <% @laundry_rooms.each do |room| %>
+<% if current_user %>
+  <h1>Your bookings:</h1>
+    <% if current_user.bookings.any? %>
+    <% current_user.bookings.each do |bookings| %>
```

Comment:
The local variable should be named in singular. It represents ONE instance of a booking, right`

Reply:
Oh, good point!